source/client-side-encryption/client-side-encryption.rst

```diff
@@ -286,13 +289,45 @@ Data keys are stored as documents in a special MongoDB collection. Data
 keys are protected with encryption by a KMS provider (AWS KMS, Azure key
 vault, GCP KMS, or a local master key).
 
+metadataClient
+^^^^^^^^^^^^^^
+A client used to run ``listCollections`` to determine whether a collection has
+an associated JSON schema with encrypted fields specified.
+
+If a ``metadataClient`` is not set, and ``bypassAutomaticEncryption=false`` then
+an internal ``MongoClient`` is used with the same options as the parent ``MongoClient``
+excluding the ``AutoEncryptionOpts``.
+
+Drivers MUST document this behavior, using the following as a template:
+
+   If the ``MongoClient`` is configured with ``AutoEncryptionOpts`` with
+   ``bypassAutomaticEncryption=false`` then a ``metadataClient`` is necessary. If a
```

Comment:
By "is necessary", does that mean drivers should throw an error if none is passed in? If so, I think that should be explicitly stated